src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsremandandsentencingapi/service/RecallService.kt

```diff
@@ -4,23 +4,41 @@ import jakarta.persistence.EntityNotFoundException
 import org.springframework.stereotype.Service
 import org.springframework.transaction.annotation.Transactional
 import uk.gov.justice.digital.hmpps.hmppsremandandsentencingapi.controller.dto.CreateRecall
-import uk.gov.justice.digital.hmpps.hmppsremandandsentencingapi.controller.dto.CreateRecallResponse
 import uk.gov.justice.digital.hmpps.hmppsremandandsentencingapi.controller.dto.Recall
+import uk.gov.justice.digital.hmpps.hmppsremandandsentencingapi.controller.dto.SaveRecallResponse
+import uk.gov.justice.digital.hmpps.hmppsremandandsentencingapi.controller.dto.UpdateRecall
 import uk.gov.justice.digital.hmpps.hmppsremandandsentencingapi.jpa.entity.RecallEntity
 import uk.gov.justice.digital.hmpps.hmppsremandandsentencingapi.jpa.repository.RecallRepository
 import java.util.UUID
 
 @Service
 class RecallService(private val recallRepository: RecallRepository) {
   @Transactional
-  fun createRecall(createRecall: CreateRecall): CreateRecallResponse {
+  fun createRecall(createRecall: CreateRecall): SaveRecallResponse {
     val recall = recallRepository.save(RecallEntity.placeholderEntity(createRecall))
-    return CreateRecallResponse.from(recall)
+    return SaveRecallResponse.from(recall)
+  }
+
+  @Transactional
+  fun updateRecall(recallUniqueIdentifier: UUID, updateRecall: UpdateRecall): SaveRecallResponse {
+    val recall = recallRepository.findOneByRecallUniqueIdentifier(recallUniqueIdentifier)
```

Comment:
the other PUT method in this API has placed the object at the UUID location if it doesn't exist rather than thrown an exception. Think its worth keeping the same behaviour for this endpoint too